packages/ai/core/generate-text/generate-text.ts

```diff
@@ -66,45 +67,6 @@ Generate a text and call tools for a given prompt using a language model.
 
 This function does not stream the output. If you want to stream the output, use `streamText` instead.
 
-@param model - The language model to use.
```

Comment:
these docs were duplicated so instead of adding to them, i rely upon the jsdoc in the options now

Reply:
maybe good for another PR, or we want to keep

Reply:
will this affect the hovers in vscode thought? prefer separate PR